README.md

```diff
@@ -33,19 +33,19 @@ Here are some features to look forward to in a future version:
 
 ## How to install?
 
-ClassTime is a web-based app that can be saved to your phone's home screen or bookmarked in your browser for easy access. 
+ClassClock is a web-based app that can be saved to your phone's home screen or bookmarked in your browser for easy access. 
 
 ### iOS
 
-1. Navigate to https://classclock.app in Safari (not tested on other browsers)
+1. Navigate to ClassClock in Safari (not tested on other browsers)
```

Comment:
We should include a link, in case people don't know the URL.

Reply:
its in the github description thingy. im thinking this part of the README will eventually be moved to like a support page or guides page or something on the classclock website